packages/web3-eth-accounts/package.json

```diff
@@ -16,7 +24,10 @@
 	"scripts": {
 		"clean": "rimraf dist && rimraf lib",
 		"prebuild": "yarn clean",
-		"build": "tsc --build",
+		"build": "yarn build:cjs && yarn build:esm && yarn build:types",
```

Comment:
We can use `npm concurrently` to run builds in parallel. what do you think?

Reply:
we can do that with & as well.